risc0/zkvm/src/tests.rs

```diff
@@ -695,7 +695,7 @@ fn pause_continue() {
 #[cfg_attr(feature = "insecure_skip_seal", ignore)]
 #[cfg_attr(feature = "cuda", serial)]
 fn continuation() {
-    let segment_limit_po2 = 16; // 64k cycles
+    let segment_limit_po2 = 17; // 128k cycles
```

Comment:
I suspect it was related to changes in risczero-wip that hadn't been propagated to risc0 (or vice versa), since it tested fine until I merged in main.

risczero-wip also needed some work to work with the new hash api. :(

I wonder if there's a way to keep risc0 and risczero-wip in sync automatically